specification/managedservices/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the ManagedServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-09
+tag: package-preview-2020-02
 ```
 
 
+### Tag: package-preview-2020-02
+
+These settings apply only when `--tag=package-preview-2020-02` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2020-02'
```

Comment:
It seems you have format the yaml for package-2019-09, but the new yaml remains no space. Meanwhile, please change it into 2020-02-preview, thanks.

Reply:
Sorry this comment did not become resolved sooner.  We also had to wait to merge these changes due to code freeze + other delays.  Please let me know if anything else needs to be resolved.